tests/integration/cheatcodes/send_message_to_l2/send_message_to_l2_test.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+from pathlib import Path
+
+from tests.integration.conftest import (
+    RunCairoTestRunnerFixture,
+    assert_cairo_test_cases,
+    CreateProtostarProjectFixture,
+)
+
+
+def test_l1_to_l2_message_cheatcode(
+    run_cairo_test_runner: RunCairoTestRunnerFixture,
+    create_protostar_project: CreateProtostarProjectFixture,
+):
+    with create_protostar_project() as protostar:
+
+        def read_contract_file(file_name: str) -> str:
+            return (Path(__file__).parent / file_name).read_text("utf-8")
+
+        protostar.create_files(
+            {
+                "tests/test_main.cairo": read_contract_file(
+                    "simple_l1_handler_test.cairo"
+                ),
+                "src/main.cairo": read_contract_file(
+                    "external_contract_with_l1_handler.cairo"
+                ),
+            }
+        )
```

Comment:
[Nit] Maybe `read_contract_file` can be merged with `ProtostarFixture::create_file` method.

```py
        protostar.create_files(
            {
                "tests/test_main.cairo": Path(__file__).parent / "simple_l1_handler_test.cairo",
            }
        )
```